packages/core/config/core-config-server-internal/tsconfig.json

```diff
@@ -13,6 +13,7 @@
     ]
   },
   "include": [
-    "src/**/*"
+    "src/**/*",
+    "@kbn/core-test-helpers-deprecations-getters"
```

Comment:
I meant you can just import this file in the code, you shouldn't need to edit the `include` array in any tsconfig.json files.

Reply:
🤦‍♀️